src/libs/PolicyUtils.ts

```diff
@@ -319,17 +319,11 @@ function getPolicyIDFromNavigationState() {
 }
 
 function getAdminEmailList(policy: Policy | null) {
-    const adminEmailList: Array<{email: string}> = [];
-    if (!policy?.employeeList) {
-        return adminEmailList;
-    }
-    Object.keys(policy.employeeList).forEach((email: string) => {
-        if (policy?.employeeList?.[email].role !== CONST.POLICY.ROLE.ADMIN) {
-            return;
-        }
-        adminEmailList.push({email});
-    });
-    return adminEmailList;
+    return Object.values(policy?.employeeList ?? {})
+        .filter((employee) => employee.role === CONST.POLICY.ROLE.ADMIN)
+        .map((employee) => ({
+            email: employee.email ?? '',
+        }));
```

Comment:
hmm not really sure - why fallback should not be defined? Probably issue in types not in way how to prepare array. Is it possible that admin user will not have specified email? But based on types from policy it could happened. That mean that we have not correct type `PolicyEmployee` @s77rt

Reply:
ok